src/main/java/io/jenkins/plugins/coverage/CoverageAction.java

```diff
@@ -21,6 +21,9 @@
     private transient WeakReference<CoverageResult> report;
     private HealthReport healthReport;
 
+    private String cacheBranchName;
+    private int branchCount = 1;
```

Comment:
I am not sure "count" is the best term for it. CC @jeffpearce

Reply:
How about `branchNumber` or `branchIndex`?

Reply:
I think `branchIndex` is good.